packages/@angular/cli/models/webpack-configs/common.ts

```diff
@@ -67,7 +67,7 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
     devtool: buildOptions.sourcemaps ? 'source-map' : false,
     resolve: {
       extensions: ['.ts', '.js'],
-      modules: [nodeModules],
+      modules: ['node_modules', nodeModules],
```

Comment:
What is the difference between `modules: ['node_modules', nodeModules],` and just `modules: ['node_modules']`?